website/docs/reference/widgets/json-form.md

```diff
@@ -18,38 +28,46 @@ To populate the JSON Form widget with data, you can utilize the **Source Data**
   "employee_id": 1001
 }
 ```
-JSON Form automatically detects the appropriate field type for each value. For instance, if the JSON data contains an `age` field, it sets the field type to a *Number Input*. Additionally, you can add/customize field types using the **Field Configuration** property.
-
-
-
-You can display dynamic data in a JSON Form widget by binding the response from a query or a JS function to the **Source Data** property. This allows the form to update dynamically as the data changes in the database or API.
-
----
-
-**Example**: suppose you want a master-detail form that shows details of each record in a form when you select a row in a table. Suppose you have fetched data from the mock database using a SELECT query `fetchUserData`. You can display the fetched data by binding the query response to the **Table Data** property of the Table widget `tblUserData`, as shown below:
 
-```js
-{{fetchUserData.data}}
-```
+Based on the JSON data provided, the JSON Form automatically identifies the appropriate field type for each value. For example, if the data contains the field `age`, the form sets the field type to a `Number Input`. Additionally, you have the flexibility to add or customize field types using the **Field Configuration property**.
```

Comment:
I think this should be talked about after you explain the dynamic data below to complete the flow. 

First - You can display dynamic data...
then add, Based on the JSON data provided.